Surface_mesh_simplification/examples/Surface_mesh_simplification/edge_collapse_bounded_normal_change.cpp

```diff
@@ -37,7 +36,7 @@ int main(int argc, char** argv)
   const std::size_t stop_n = (argc > 2) ? std::stoi(argv[2]) : num_halfedges(surface_mesh)/2 - 1;
   SMS::Count_stop_predicate<Surface_mesh> stop(stop_n);
 
-  typedef SMS::Bounded_normal_change_placement<SMS::LindstromTurk_placement<Surface_mesh> > Placement;
+  typedef SMS::LindstromTurk_placement<Surface_mesh> Placement;
```

Comment:
You can even use the default cost / placement now; it needed to be explicit because you had to wrap, but now this example is just about using the named parameter.